### PR TITLE
Add tensorflow demo

### PR DESCRIPTION
**Pull request by ClawSeven**

### CODE REVIEW

File: demos/tensorflow/tensorflow_training/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## How to Run
+
+This tutorial is written under the assumption that you have Docker installed and use Occlum in a Docker container.
+
+Occlum is compatible with glibc-supported Python, we employ miniconda as python installation tool. You can import TensorFlow packages using conda. Here, miniconda is automatically installed by install_python_with_conda.sh script, the required python and TensorFlow package and MNIST dataset for this project are also loaded by this script. 
+
+Step 1 (on the host): Start an Occlum container
+```
+docker pull occlum/occlum:0.23.0-ubuntu18.04
```
[the remark this answers is not in the document]
Comment (ClawSeven) [reply demos/tensorflow/tensorflow_training/README.md:21]: Done

File: demos/tensorflow/tensorflow_training/install_python_with_conda.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+set -e
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd )"
+
+# Install python and dependencies
+[ -f Miniconda3-latest-Linux-x86_64.sh ] || wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+[ -D miniconda ] || bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $script_dir/miniconda
+$script_dir/miniconda/bin/conda create --prefix $script_dir/python-occlum -y  python=3.7 tensorflow==1.15.0
+
+# Download mnist dataset
+wget https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz
```
[the remark this answers is not in the document]
Comment (ClawSeven) [reply demos/tensorflow/tensorflow_training/install_python_with_conda.sh:11]: Done

File: demos/tensorflow/tensorflow_training/install_python_with_conda.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+set -e
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd )"
+
+# Install python and dependencies
+[ -f Miniconda3-latest-Linux-x86_64.sh ] || wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+[ -d miniconda ] || bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $script_dir/miniconda
+$script_dir/miniconda/bin/conda create --prefix $script_dir/python-occlum -y  python=3.7 tensorflow==1.15.0
+
+# Download mnist dataset
+[ -f mnist.npz ] wget https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz
```
Comment (guzongmin) [inline demos/tensorflow/tensorflow_training/install_python_with_conda.sh:11]: Is it correct?
[ -f mnist.npz ] || wget https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**